torchdata/datapipes/iter/load/aisio.py

```diff
@@ -17,19 +15,37 @@
     from aistore.pytorch.utils import parse_url, unparse_url
 
     HAS_AIS = True
+
 except ImportError:
     HAS_AIS = False
 
+try:
+    import aistore
+    from packaging.version import parse as parse_version
+
+    AIS_VERSION_CHECK = parse_version(aistore.__version__) >= parse_version("1.0.2")
+
+except (ImportError, AttributeError):
+    AIS_VERSION_CHECK = False
+
 
 def _assert_aistore() -> None:
     if not HAS_AIS:
         raise ModuleNotFoundError(
-            "Package `aistore` is required to be installed to use this datapipe."
-            "Please run `pip install aistore` or `conda install aistore` to install the package"
+            "Package `aistore` (>=1.0.2) is required to be installed to use this datapipe."
```

Comment:
```suggestion
            "Package `aistore` (==1.0.2) is required to be installed to use this datapipe."
```
nit: to be consistent with `requirements_aistore.txt`. Let me know if there is a reason to be different.